src/components/Article.jsx

```diff
@@ -1,3 +1,65 @@
+import { useParams } from "react-router-dom"
+import {getArticleById, patchArticleById} from '../utils/API'
+import { useState,useEffect } from "react"
+import Like from '../icons/thumb-up.png'
+import dislike from '../icons/dislike.png'
+import '../styles/Article.css'
+import { Link } from "react-router-dom"
+
 export const Article = () =>{
-    
+    const {article_id} = useParams()
+    const [article,setArticle] = useState({})
+    const [like , setLike] = useState(0)
+    const [undo, setUndo] = useState(0)
+
+    useEffect(()=>{
+        getArticleById(article_id).then(article=>{
+            setArticle(article)})
+    },[article_id])
+
+    useEffect(()=>{
+        patchArticleById(article_id,like)
+    },[like])
+
+    useEffect(()=>{
```

Comment:
Is this for up/downvoting? I think you can do this all in one useeffect that takes something like a 'direction' and can be handled either way.

Reply:
I tried doing it in one but couldn't figure it out. ended up needing the second for the conditional rendering of the buttons. Not sure what 'direction' means sorry.